backend/src/main/kotlin/hu/bme/sch/cmsch/component/token/TokenAdminTokensByUsersOfGroupsController.kt

```diff
@@ -116,27 +116,26 @@ class TokenAdminTokensByUsersOfGroupsController(
         val document = Document(pdfDoc, PageSize.A4)
         document.setMargins(20f, 40f, 20f, 40f)
 
-        val kirdevLogo: Image = Image(ImageDataFactory.create(
-            ClassPathResource("/static/images/kirdev-logo.png").inputStream.readAllBytes()
-        )).scaleToFit(70f, 70f)
-
-        val ssslLogo: Image = Image(ImageDataFactory.create(
-            ClassPathResource("/static/images/sssl-logo.png").inputStream.readAllBytes()
-        )).scaleToFit(70f, 70f)
-
         val font = PdfFontFactory.createFont("OpenSans-Regular.ttf")
         val header = Paragraph()
-            .add(ssslLogo)
-            // FIXME: place it into a setting
-            .add(Paragraph("GÓLYAKÖRTE 2023\nJELENLÉTI - ${group.name}")
+        readAsset(tokenComponent.reportLogo.getValue().replace("/cdn/", "/")).map {
+            Image(ImageDataFactory.create(it,)).scaleToFit(70f, 70f)
+        }.ifPresent(header::add)
+
+        val eventName = tokenComponent.reportTitle.getValue()
+        header.add(Paragraph("${eventName}\nJELENLÉTI - ${group.name}")
                 .setTextAlignment(TextAlignment.CENTER)
                 .setVerticalAlignment(VerticalAlignment.MIDDLE)
                 .setFont(font)
                 .setFontSize(28f)
                 .setMarginLeft(40f)
                 .setMarginRight(40f))
-            .add(kirdevLogo)
-            .setTextAlignment(TextAlignment.CENTER)
+
+        readAsset("/static/images/kirdev-logo.png").map {
```

Comment:
Maybe make it optional using a boolean config variable? I'm not really sure.

Reply:
No no, we must show the kir-dev logo at every opportunity